src/components/VoteScreen.js

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import axios from 'axios';
+
+import CandidateStatus from './CandidateStatus.js';
+
+export default function VoteScreen() {
+  const [candidates, setCandidates] = useState(null);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await axios.get(
+          'http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/candidates'
+        );
+        setCandidates(response.data);
```

Comment:
```suggestion
        const {data}= await axios.get(
          'http://ec2-3-34-5-220.ap-northeast-2.compute.amazonaws.com:2020/candidates'
        );
        setCandidates(data);
```
이렇게 destructuring해봐도 좋을 것 같습니당